test/xexp.jl

```diff
@@ -0,0 +1,39 @@
+@testset "xexp" begin
+    @test isnan(xexp(NaN))
+    @test isinf(xexp(Inf))
+    @test xexp(-Inf) == 0
+    @test xexp(0.0) == 1
+
+    @test isinf(xexp(710.0))
+    @test xexp(-746.0) == 0
+
+    x = linspace(708.4, 709.7, 25)
+    @test_approx_eq xexp.(x) exp.(x)
```

Comment:
Totally up to you but what I would do personally is format it as
```julia
let x = linspace(708.4, 709.7, 25)
    xexp.(x) ≈ exp.(x)
end
```
That has the benefit of scoping `x` so you aren't overwriting it for each test, though it doesn't really matter much. Just a stylistic preference.

That said, we really should be using `≈`, as Base has switched away from `@test_approx_eq` in favor of it.